src/styles.css/reset.css

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+.page__body,
+.title,
+.text {
```

Comment:
since in reset.css you want to reset default tag styles here is better to use tag selectors: `body, h2, p`